modules/olm-installing-from-operatorhub-using-cli.adoc

```diff
@@ -131,3 +131,183 @@ $ oc apply -f sub.yaml
 ----
 +
 At this point, OLM is now aware of the selected Operator. A cluster service version (CSV) for the Operator should appear in the target namespace, and APIs provided by the Operator should be available for creation.
+
+. As a cluster administrator, you can configure the following components for the Operator:
```

Comment:
"As a cluster administrator"
* This is understood, and can be removed.

Are these fields optional? If they are, consider whether you can label this step as `Optional`.

Reply:
Yes, these steps are optional. CC: @bandrade Please correct me if I am wrong. 

Ack- removed "As a cluster administrator"

Reply:
> In its current structure, users create a subscription object in steps 4 and 5, and then they create (or overwrite?) a series of other subscription resources, each with a different field that you describe.
> 
The idea is to create a series of other resources.

> I wonder whether it could be more useful instead to add this information to the sample YAML configuration in Step 4. You could perhaps add each of the new fields there and describe them in the callouts.
> 
> That way, users can know how to create the complete subscription resource all in one go.
> 
> What do you think?
> 
I agree. I will restructure it in that way.

> CC @adellape

Reply:
Addressed